src/Entity/ElectedRepresentative/ElectedRepresentative.php

```diff
@@ -188,6 +203,9 @@ public function __construct(
         $this->officialId = $officialId;
         $this->socialNetworkLinks = new ArrayCollection();
         $this->mandates = new ArrayCollection();
+        $this->sponsorships = new ArrayCollection();
+
+        $this->createSponsorships();
```

Comment:
why do you need to initialize sponsorship collection?

Reply:
I added it to have the lines for 4 years when we edit a sponsorship, here is the specification: https://app.clubhouse.io/enmarche/story/10957/cr%C3%A9ation-du-sous-module-parrainages-sur-la-fiche-%C3%A9lu